lib/tasks/populate.rake

```diff
@@ -0,0 +1,38 @@
+desc "Populate the database with a course, assignment, problem and test"
+task populate:
```

Comment:
Can you add a `db` namespace to this. So instead of `rake populate`, we can use `rake db:populate`
```
namespace :db do
  desc "This task does nothing"
  task :nothing do
    # NOTHING
  end
end
--



```